README.md

```diff
@@ -0,0 +1,370 @@
+
+An authorized reviewer will initiate contact verification by sending each security contact a PGP-encrypted email containing random words.
+You will be asked to post the contents of these mails in your `shim-review` issue to prove ownership of the email addresses and PGP keys.
+*******************************************************************************
+- Name: Jason Rodriguez
+- Position: Engineer
+- Email address: jrodriguez@ciq.com
+- PGP key fingerprint: 0310 CFD4 0447 4D14 5072 D3E1 EAFF ECB3 C3AB C924
+- PGP key available on keys.openpgp.org
+- https://keys.openpgp.org/vks/v1/by-fingerprint/0310CFD404474D145072D3E1EAFFECB3C3ABC924
+
+(Key should be signed by the other security contacts, pushed to a keyserver
+like keyserver.ubuntu.com, and preferably have signatures that are reasonably
+well known in the Linux community.)
+
+*******************************************************************************
+### Who is the secondary contact for security updates, etc.?
+*******************************************************************************
+- Name: Michael Young
+- Position: Information Technology Director
+- Email address: myoung@ciq.com
+- PGP key fingerprint:  CD82 9808 7BCA C022 B5EC  84FA D84A 6A59 1392 6D2B
+- http://keyserver.ubuntu.com/pks/lookup?op=get&search=0xcd8298087bcac022b5ec84fad84a6a5913926d2b
+
+(Key should be signed by the other security contacts, pushed to a keyserver
+like keyserver.ubuntu.com, and preferably have signatures that are reasonably
+well known in the Linux community.)
+
+*******************************************************************************
+### Were these binaries created from the 15.8 shim release tar?
+Please create your shim binaries starting with the 15.8 shim release tar file: https://github.com/rhboot/shim/releases/download/15.8/shim-15.8.tar.bz2
+
+This matches https://github.com/rhboot/shim/releases/tag/15.8 and contains the appropriate gnu-efi source.
+
+******************************************************************************
+Yes. no other patches are applied
+
+
+*******************************************************************************
+### URL for a repo that contains the exact code which was built to get this binary:
+*******************************************************************************
+CIQ shim-unsigned-x64 RPM repository:  [https://www.github.com/ctrliq/shim-unsigned-x64/src/ciq9/](https://github.com/ctrliq/shim-unsigned-x64/tree/ciq9)
+
+This code is a combination of:  https://github.com/rhboot/shim/releases/download/15.8/shim-15.8.tar.bz2 and an RPM spec file derived from the Rocky (and in turn RHEL) one.
+The Build uses archives of Rocky 9.2 dependencies: 
+baseurl=https://dl.rockylinux.org/vault/rocky/9.2/AppStream/x86_64/os/
```

Comment:
Again, aarch64 vs x86_64?

Reply:
Good catch.  Should be `https://dl.rockylinux.org/vault/rocky/9.2/AppStream/aarch64/os/`